plugins/woocommerce/client/admin/client/settings-payments/settings-payments-bacs.tsx

```diff
@@ -0,0 +1,182 @@
+/**
+ * External dependencies
+ */
+import {
+	Card,
+	CardBody,
+	CheckboxControl,
+	TextControl,
+	TextareaControl,
+	Button,
+} from '@wordpress/components';
+import { useState, useEffect } from 'react';
+import { __ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+import {
+	PAYMENT_GATEWAYS_STORE_NAME,
+	type PaymentSelectors,
+} from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import './settings-payments-body.scss';
+import './settings-payments-form.scss';
```

Comment:
It seems like we're importing these styles in multiple files. This means they are being loaded multiple times. It would be great if we can optimize this.